packages/stark-ui/src/modules/app-data/components/app-data.component.ts

```diff
@@ -35,15 +24,11 @@ export type StarkAppDataComponentMode = "dropdown" | "menu";
 		class: componentName
 	}
 })
-export class StarkAppDataComponent extends AbstractStarkUiComponent implements OnInit, OnDestroy {
+export class StarkAppDataComponent extends AbstractStarkUiComponent implements OnInit {
 	/**
 	 * The mode in which the component should be displayed.
 	 */
-	@Input() public mode?: StarkAppDataComponentMode = "dropdown";
-
-	public isDetailHidden = true;
-
-	public windowClickHandler?: EventListener;
+	@Input() public mode: StarkAppDataComponentMode = "dropdown";
```

Comment:
This input should be kept as optional cause the "dropdown" mode is the default. Otherwise, the developer will need to specify this always. So I think we just need to add the following to the comment of this input: `Default: "dropdown"`

Reply:
Thanks for the explanation, I didn't remember why it was optional